src/main/java/org/primefaces/model/filter/GlobalFilterConstraint.java

```diff
@@ -23,16 +23,9 @@
  */
 package org.primefaces.model.filter;
 
-import javax.faces.context.FacesContext;
-import java.util.Locale;
-
 public class GlobalFilterConstraint extends ContainsFilterConstraint {
 
     public GlobalFilterConstraint() {
     }
 
-    @Override
-    public boolean isMatching(FacesContext ctxt, Object value, Object filter, Locale locale) {
-        return super.isMatching(ctxt, value, filter, locale);
-    }
```

Comment:
Sonar recommended removing this method as its just overriding and calling SUPER.  Removed and verified all integration tests are still working.